--- origsrc/liboggz-0.9.5/src/liboggz/Makefile.am	2005-09-06 01:27:27.000000000 -0500
+++ src/liboggz-0.9.5/src/liboggz/Makefile.am	2006-06-05 20:43:30.250000000 -0500
@@ -21,5 +21,5 @@
 	oggz_vector.c oggz_vector.h \
 	metric_internal.c
 
-liboggz_la_LDFLAGS = -version-info @SHARED_VERSION_INFO@ @SHLIB_VERSION_ARG@
+liboggz_la_LDFLAGS = -version-info @SHARED_VERSION_INFO@ @SHLIB_VERSION_ARG@ -no-undefined
 liboggz_la_LIBADD = @OGG_LIBS@
